resources.go

```diff
@@ -2,13 +2,29 @@ package render
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
 	"net/http"
 )
 
+type ResourceGetEnvOptions struct {
+	Cursor string `url:"cursor"`
+	Limit  string `url:"limit"`
```

Comment:
According to docs, this should be an int

Reply:
Good catch. Fixed.